### DISCUSSION TITLE
1.4.0rc1

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#17528] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.4.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.4.0rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.4.0rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.4.0rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.4.0rc1/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.4.0rc1'>1.4.0rc1</a>.</em>

**Comment by kamalkeshavani-aiinside** [comment]
Request to add these PRs as well in 1.4.0:
- apache/superset#17341
- apache/superset#17420

**Reply by kamalkeshavani-aiinside** [reply]
One more:
- apache/superset#17416

**Reply by kamalkeshavani-aiinside** [reply]
And another:
- apache/superset#17615

**Comment by kamalkeshavani-aiinside** [comment]
Not sure if this issue was in 1.3. Confirmed not in 1.2.
Histogram tooltip is showing a formula for percentile calculation instead of the result value.
![image](https://user-images.githubusercontent.com/74634977/143188255-1bb60497-9235-47d8-bfb9-89e020f54a73.png)

**Reply by villebro** [reply]
@kamalkeshavani-aiinside can you check if this is still reproducible on master, and if so, open an issue for it so we can get it fixed and potentially get it cherried into a patch release?

**Reply by kamalkeshavani-aiinside** [reply]
@villebro Confirmed the issue on master. Created an issue https://github.com/apache/superset/issues/17550

**Comment by kamalkeshavani-aiinside** [comment]
Probably not all will be solved so quickly, but still taking my chances here. Please check if some of these can be fixed in 1.4:
Bugs:
- apache/superset#15674
- apache/superset#16658

Feature Requests:
- apache/superset#17211
- apache/superset#17219
- apache/superset#16159
- apache/superset#15672

**Comment by rusackas** [comment]
This ship has sailed, so I'll close the discussion. DM me on [Superset Slack](http://bit.ly/join-superset-slack)  if there's reason to re-open it.
